test/fixtures.ts

```diff
@@ -36,16 +36,26 @@ const js = {
 			];
 		})();
 
-		export const esnext = (() => {
+		export const es2021 = (() => {
 			let a;
+			let x = 1;
+			const y = 3;
+			x &&= y;
+			return [
+				a ??= 2,
+				x,
+				1_000_000_000,
+			];
+		})();
+
+		export const esnext = (() => {
 			return [
 				class { x = 2; },
 				class { static x = 1; },
 				class { #x() {} },
 				class { #x },
-				class { static #x() {} },
+				// class { static #x() {} },
```

Comment:
Why was this commented out? There should still be support for this as per [esbuild docs](https://esbuild.github.io/content-types/#javascript).

Reply:
I don't know it generates this error

```ts
    evalmachine.<anonymous>:86
      if (member.has(obj))
                 ^

    TypeError: Cannot read property 'has' of undefined
```

I'll dig into it at the end of the week

Reply:
Looks good to me. Thank you!

Reply:
Thank you for reporting this bug upstream. And Evan already made a patched release for this (🚀)
I guess we can put back the original `class { static #x() {} }` in the array with [esbuild v0.12.18](https://github.com/evanw/esbuild/commit/88da5edebb4efb7d1639b707c9e3ae02b89e9a32)